src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -44,6 +44,7 @@
     <div class="controls-flex">
       <h4>Misc. Controls</h4>
       <ToggleButton
+        id="arm_laser"
```

Comment:
When I was trying to add toggle buttons to the carousel component, I did it without an ID, but this messed with the Raman laser toggle since it also didn't have an ID. Basically, the carousel callback was overwritten by the Raman callback. Adding ids fixed this since it makes the callbacks unique, so I went ahead and added IDs to all toggle buttons so we don't run into this again in the future